app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -48,10 +49,20 @@ class TSnePlotViewRoute extends React.Component {
   }
 
   render() {
+
+    console.log(this.props)
+
     const { location, match, history } = this.props
     const { atlasUrl, suggesterEndpoint, defaultPlotMethodAndParameterisation } = this.props
     const { species, experimentAccession, accessKey, ks, ksWithMarkerGenes, plotTypesAndOptions, metadata, anatomogram } = this.props
     const search = URI(location.search).search(true)
+    if(species === `homo sapiens` && Object.keys(anatomogram).length >0){
```

Comment:
I don't understand this condition. Currently we have anatomograms only for `homo sapiens`, but we are going to have them for other species in the very near future.
Do we need this check at all?
Thanks